typescript/ica/index.ts

```diff
@@ -0,0 +1,10 @@
+export {
```

Comment:
Why is this stuff in a separate packages instead of being included in the SDK?

Reply:
good question, I guess it can be bundled in

Reply:
will migrate in when working on ICA SDK ticket